src/main/java/com/minecolonies/coremod/entity/ai/citizen/baker/EntityAIWorkBaker.java

```diff
@@ -8,14 +9,12 @@
 import com.minecolonies.coremod.entity.ai.basic.AbstractEntityAISkill;
 import com.minecolonies.coremod.entity.ai.util.AIState;
 import com.minecolonies.coremod.entity.ai.util.AITarget;
-import com.minecolonies.coremod.entity.ai.util.RecipeStorage;
+import com.minecolonies.coremod.placementhandlers.PlacementHandlers;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.placementhandlers.PlacementHandlers'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)